.agents/skills/create-api-endpoint/SKILL.md

```
@@ -0,0 +1,365 @@
+
+# Workflow (6 steps)
+
+## Step 1 — Parse the request, validate the feature
+
+1. Extract from the user's input:
+   - HTTP method (`GET / POST / PUT / PATCH / DELETE`)
+   - Path (e.g., `/callvan/posts/{postId}/ban`)
+   - Feature hint (usually the first path segment: `callvan`)
+2. Normalize feature: lowercase = `callvan`, PascalCase = `Callvan`.
+3. Run these Greps in parallel to map existing files:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)